components/layout/checkAccount/intex.ts

```diff
@@ -0,0 +1 @@
+export * from "./check-account-form";
```

Comment:
Did you mean `components/layout/checkAccount/index.ts`?

Reply:
ah ya, thanks for the correction